doc/queries.md

```diff
@@ -871,6 +871,44 @@ Result:
 }
 ```
 
+### Tiered Access Sensitive Record Exclusion
+It is possible to configure Guppy to hide some records from being returned in `_aggregation` queries when Tiered Access is enabled (tierAccessLevel: "regular").
+The purpose of this is to "hide" information about certain sensitive resources, essentially making this an escape hatch from Tiered Access. Specifically, this feature is used in a Gen3 data commons to hide the existence of some studies from clients who do not have access, while keeping the features of Tiered Access for the non-sensitive studies.
+
+Crucially, Sensitive Record Exclusion only applies to records which the user does not have access to. If the user has access to a record, it will
+be counted in the aggregation query whether or not it is sensitive.
+
+To enable Sensitive Record Exclusion, the `tierAccessSensitiveRecordExclusionField` value should be set in Guppy's config. The value of the `tierAccessSensitiveRecordExclusionField` should match a boolean field in the Elasticsearch index that indicates whether or not a record is sensitive.
+(E.g., `tierAccessSensitiveRecordExclusionField: "sensitive"` in the Guppy config tells Guppy to look for a field in the ES index called `sensitive`, and to exclude records in the ES index which have `sensitive: "true"`)
```

Comment:
this line and the above line could be also put in README.md [here](https://github.com/uc-cdis/guppy#tier-access) and it should be `tier_access_sensitive_record_exclusion_field` instead of camelCase style.

Reply:
Thank you, good catch!!